### ISSUE TITLE
About Training

### ISSUE THREAD

**Issue by Pluto1314** [ifzhang/FairMOT#6]
Hi, I think this is a nice work.
I have a question, if I only use the MOT17 dataset to continue training the model, can I create a ft_mot17_dla34.sh like ft_mot15_dla34.sh?
At the same time,
cd src
python train.py mot --exp_id ft_mot15_dla34 --gpus 0,1 --batch_size 8 --load_model '../models/all_dla34.pth' --data_cfg '../src/lib/cfg/mot15.json'- -num_epochs 10 --lr 1e-5
cd ..
The content will be changed to:
cd src
python train.py mot --exp_id ft_mot17_dla34 --gpus 0,1 --batch_size 8 --load_model '../models/all_dla34.pth' --data_cfg '../src/lib/cfg/mot17.json'- -num_epochs 10 --lr 1e-5
cd ..

Is my reformation correct? If it is not correct, how will I revise it?
Can you give me some suggestions? Thank you!

**Comment by ifzhang** [comment]
Yes, you are correct! The training dataset of the baseline model already contains MOT17, so if you continue training the model on MOT17, you can see the loss is very low. Also, the results will be similar to the baseline model.

**Comment by Pluto1314** [comment]
ok, thanks!
